server/routes/serviceProviderReferrals/postSessionBehaviourFeedbackPresenter.test.ts

```diff
@@ -14,14 +14,84 @@ describe(PostSessionBehaviourFeedbackPresenter, () => {
         behaviourDescription: {
           question: `Describe Alex's behaviour in this session`,
           hint: 'For example, consider how well-engaged they were and what their body language was like.',
+          errorMessage: null,
```

Comment:
We've normally been nesting these inside the `fields` object like this:

```
behaviourDescription: {
  value: 'foo',
  errorMessage: 'blah'
}
```

— could we keep this consistent?

Reply:
Just looking at `rarDaysPresenter`, `actionPlanNumberOfSessionsPresenter` and `riskInformationPresenter` they're nested in `text` - can you show me an example where they're nested under `fields`?

Reply:
Yeah, it might have been different in the past. But recently, I added some new methods to `PresenterUtils`, which return a "field presenter" object which contains both the value and the error message. See them in use in `EditSessionPresenter`.

Reply:
have updated this to how I _think_ you mean